charts/gardener-extension-provider-azure/templates/deployment.yaml

```diff
@@ -55,9 +55,7 @@ spec:
         - --disable-controllers={{ .Values.disableControllers | join "," }}
         - --disable-webhooks={{ .Values.disableWebhooks | join "," }}
         - --health-bind-address=:{{ .Values.healthPort }}
-        {{- if .Values.gardener.version }}
```

Comment:
I forgot to clean this up in https://github.com/gardener/gardener-extension-provider-aws/pull/538.
I will perform this for provider-aws as well.

Reply:
Done in https://github.com/gardener/gardener-extension-provider-aws/pull/542